package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "npx tsc",
+    "build": "npx tsc && rm -rf dist/templates && cp -r src/templates dist/templates",
```

Comment:
not sure if there is a better way to do this, but I figured this is at least a quick and dirty way to pull templates into the `dist` folder.

Reply:
I personally used a npm package called `typescript-cp` for it and then just using the watch flag in dev and without it when building it.

Reply:
👌